packages/x-components/src/adapter/mocked-adapter.ts

```diff
@@ -71,7 +72,10 @@ class E2ETestsAdapter extends EmpathyAdapter {
   }
 
   track(request: TrackingRequest): Promise<void> {
-    return mockFetch(request, 'track');
+    return fetch(request.url, {
+      method: 'POST',
+      body: JSON.stringify(request.params)
+    }).then(noOp);
```

Comment:
why the `.then(noOp)`?

Reply:
`SearchAdapter` specifies that `track` method should return `Promise<void>`, but `fetch` returns a `Promise<Response>`. With the `noOp` we are simply ignoring that `Response` and "transforming" it into `void`